ha/alert/alert_filter_exceptions.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
```

Comment:
filename can we use like
alert_exception.py

so that all alert exception can be recorded here

Reply:
Done